android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -112,6 +113,15 @@ protected void doFrameGuarded(long frameTimeNanos) {
     mNoopNode = new NoopNode(this);
   }
 
+  public static NodesManager getInstanceWithContext(ReactContext context){
+    if(sInstance == null || sInstance.mContext != context) {
```

Comment:
I'm not sure if we should recreate instance if the reference to `ReactContext` changed